api/config/middleware.js

```diff
@@ -0,0 +1,12 @@
+const cors = require('cors');
+const express = require('express');
+const userRouter = require('../RESTful API/Users/userRouter');
+const postRouter = require('../RESTful API/Posts/postRouter');
+
+module.exports = (server) => {
+	server.use(cors());
+	server.use(express.json());
+
+	server.use('/api/user', userRouter);
```

Comment:
The general idea is that route leading to a "collection" of things should have plural name so `api/users` would be better choice.